Source/Scene/ModelExperimental/GeometryPipelineStage.js

```diff
@@ -177,12 +177,17 @@ GeometryPipelineStage.process = function (
     }
     //>>includeEnd('debug');
 
-    // Classification models only use the position and feature ID attributes.
+    // Classification models only use the position, texcoord, and feature ID attributes.
     const isPositionAttribute =
       attribute.semantic === VertexAttributeSemantic.POSITION;
     const isFeatureIdAttribute =
       attribute.semantic === VertexAttributeSemantic.FEATURE_ID;
-    if (hasClassification && !isPositionAttribute && !isFeatureIdAttribute) {
+    const isTexcoordAttribute =
+      attribute.semantic === VertexAttributeSemantic.TEXCOORD;
```

Comment:
Does the `DequantizationPipelineStage` also need to be updated?

Reply:
thanks for the catch!